src/worker.py

```
@@ -1697,6 +1697,301 @@ def _persist_attachment(self, session_id, info):
+                current["hand_raised"] = next_hand
+                delta["hand_raised"] = next_hand
+                changed = True
+
+        if not changed:
+            return
+
+        self.presence[user_id] = current
+        self._persist_user_attachments(user_id)
+        self._broadcast(json.dumps(delta), exclude_session_id=sid)
+
+    async def on_webSocketClose(self, ws, code, reason, wasClean):
```

Ruff (0.15.10)

[warning] Unused method argument: `code`

(ARG002)